samples/JSON/filenames/.all-contributorsrc

```diff
@@ -0,0 +1,923 @@
+{
+  "projectName": "all-contributors",
+  "projectOwner": "all-contributors",
+  "files": [
+    "README.md"
+  ],
+  "imageSize": 100,
+  "contributors": [
+    {
+      "login": "kentcdodds",
+      "name": "Kent C. Dodds",
+      "avatar_url": "https://avatars.githubusercontent.com/u/1500684?v=3",
+      "profile": "https://kentcdodds.com",
+      "contributions": [
+        "question",
+        "doc",
+        "review",
+        "talk"
+      ]
```

Comment:
I think we can trim the `contributors` array down to only a few entries. The entries are mostly homogenous, so our classifier isn't really benefiting from the repetition.

Reply:
Wow, thanks for the quick response! I was a little unsure about this because, while certainly real-world, it did feel overboard. So would you say about 3 would be enough?

Reply:
Yep, 3's plenty. 👍